tests/fixtures/dnsimple-invalid-content.json

```diff
@@ -1,106 +0,0 @@
-{
-  "data": [
-    {
-      "id": 11189898,
-      "zone_id": "unit.tests",
-      "parent_id": null,
-      "name": "naptr",
-      "content": "",
```

Comment:
Hrm. Wonder if i managed to get empty content into/out of their provider or if I was just aggressively testing things. Either way most of these seem like cases that are now handled globally. Only one I'm not sure about the naptr. Looks like that was converting to an empty value which was just weird in general. All of that predated the validation improvements so it's probably fine for them to go away so long as the code coverage is still complete.

Reply:
Looks like there's some missing coverage in `DnsimpleProvider` now for some special cases where they allow invalid values to be submitted to their API for the complex record types (NAPTR, SSHFP, SRV.) Those snippets are similar to the special case needed for NS1 in your other PR. 

Played around with it a bit and it looks like they're covered correctly if this fixture and the tests below are restored and a few `lenient`s are added to the `populate` and record creations in the test code below.

```python
+            provider.populate(zone, lenient=True)
+            self.assertEquals(set([
+                Record.new(zone, '', {
+                    'ttl': 3600,
+                    'type': 'SSHFP',
+                    'values': []
+                }, lenient=True),
+                Record.new(zone, '_srv._tcp', {
+                    'ttl': 600,
+                    'type': 'SRV',
+                    'values': []
+                }, lenient=True),
+                Record.new(zone, 'naptr', {
+                    'ttl': 600,
+                    'type': 'NAPTR',
+                    'values': []
+                }, lenient=True),
+            ]), zone.records)
```